### PR TITLE
Add Katello 3.8 pipeline

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by zjhuntin** [comment]
ACK

**Comment by ehelms** [comment]
@zjhuntin Mind expanding on your concern?

File: playbooks/setup_forklift.yml [hunk 1]
```diff
@@ -22,5 +25,6 @@
         state: 'present'
     - name: 'Clone Forklift'
       git:
-        repo: 'http://github.com/theforeman/forklift'
+        repo: "http://github.com/{{ github_org }}/forklift"
```
Comment (ekohl) [inline playbooks/setup_forklift.yml]: I'd just use `forklift_url` so you're not tied to github. You can also change it to https://

Comment (ehelms) [reply playbooks/setup_forklift.yml]: Where else would I get than from github?

Comment (ekohl) [reply playbooks/setup_forklift.yml]: I imagine there could be instances where you have a local mirror. I also think `forklift_url` is a better naming than `github_org`.

Comment (evgeni) [reply playbooks/setup_forklift.yml]: We have already something similar in the `bats` role:

https://github.com/theforeman/forklift/blob/96a04c3e180ee8794dfa7df30602ae6126c0caa0/roles/bats/tasks/main.yml#L33-L38

should that be moved out to an own role instead?

Comment (evgeni) [reply playbooks/setup_forklift.yml]: (and strictly speaking, this does not belong into this PR anyways ;))

**Comment by ehelms** [comment]
Updated and switch to forklift_url

File: playbooks/setup_forklift.yml [hunk 2]
```diff
@@ -11,16 +11,22 @@
 
 - hosts: all
   become: true
+  vars:
+    forklift_url: "http://github.com/theforeman/forklift"
```
Comment (ekohl) [inline playbooks/setup_forklift.yml]: Could you switch this to https to avoid the redirect?

File: pipelines/pipeline_katello_38.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    - forklift
+
+- hosts:
+  - pipeline-katello-3.8-centos7
+  - pipeline-proxy-3.8-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 4
+    katello_repositories_version: 3.8
+    foreman_repositories_version: 1.18
```
Comment (ekohl) [inline pipelines/pipeline_katello_38.yml]: 3.8 is 1.19

**Comment by ehelms** [comment]
Updated

**Comment by evgeni** [comment]
IMHO this looks good. I'd role-out the forklift stuff in another PR.

@ekohl as you were raising concerns, please have a final ack :)

File: pipelines/pipeline_katello_38.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        memory: 3072
+  roles:
+    - forklift
+
+- hosts:
+  - pipeline-katello-3.8-centos7
+  - pipeline-proxy-3.8-centos7
+  become: yes
+  vars:
+    puppet_repositories_version: 4
```
Comment (evgeni) [inline pipelines/pipeline_katello_38.yml]: Oh, btw, should this be 5 these days?

**Comment by ehelms** [comment]
@evgeni Good catch, updated

**Pull request merged**